### PR TITLE
Support bulk deletes in auth users

### PR DESCRIPTION
**Pull request by joshenlim**
image

### CODE REVIEW

File: apps/studio/components/interfaces/Auth/Users/InviteUserModal.tsx [hunk 1]
```diff
@@ -44,52 +44,49 @@ const InviteUserModal = ({ visible, setVisible }: InviteUserModalProps) => {
   }
 
   return (
-    <div>
```
Comment (joshenlim) [inline apps/studio/components/interfaces/Auth/Users/InviteUserModal.tsx:47]: just removed an unnecessary div here

**Review by alaister — COMMENTED** [review]
If you delete all your users at once, the side panel seems to stay open

Screenshot 2024-12-20 at 21 18 08

**Comment by joshenlim** [comment]
Oh good catch! Should close the panel if the user thats in view is deleted 🙏

**Comment by joshenlim** [comment]
@alaister just pushed a fix up! 🙂 🙏

**Review by alaister — APPROVED** [review]
LGTM!

- Tested on preview

**Review by alaister — APPROVED** [review]

**Pull request merged**